### PR TITLE
SQL Basics: Solutions

### PR DESCRIPTION
**Pull request by akabishau**
This PR marks the completion of the assignment for Lesson "SQL Basics" of the Section 1. I used the .sql file extension instead of .txt to have SQL code formatting. I hope that's okay.

### CODE REVIEW

File: lesson-1.sql [hunk 1]
```diff
@@ -1,19 +1,36 @@
 --(1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 --Use LIMIT and OFFSET to get entries 11 through 20. 
---Paste your SQL statement below. 
+--Paste your SQL statement below.
+select CustomerID, CustomerName, ContactName, Country 
+from Customers
+limit 10 offset 10
```
Comment (josiahandrew12) [inline lesson-1.sql:6]: So you will also need to `ORDER BY Country `

**Comment by josiahandrew12** [comment]
@akabishau One small comment but everything else looks good.

File: lesson-1.sql (hunk 1 above)
Comment (akabishau) [reply lesson-1.sql:7]: Missed it. Updated

**Comment by akabishau** [comment]
@josiahandrew12, did you validated the last changes I made? I can't find the email with Successful Completion of the assignment 1.1. Apologies, if I missed the email.

**Review by josiahandrew12 — APPROVED** [review]

**Pull request merged**